src/view/view-impl.cpp

```diff
@@ -227,7 +227,7 @@ wlr_surface *wf::wlr_view_t::get_keyboard_focus_surface()
 
 bool wf::wlr_view_t::should_be_decorated()
 {
-    return role == wf::VIEW_ROLE_TOPLEVEL && !has_client_decoration;
+    return role == wf::VIEW_ROLE_TOPLEVEL && !has_client_decoration && !fullscreen;
```

Comment:
the decoration plugin is responsible for this, dont' put it here. If you see some bug related to this, let's look into how to fix it in the decoration plugin.

Reply:
fair enough, will remove and we'll fix whatever happens